testing/pytest-xdist/APKBUILD

```diff
@@ -0,0 +1,54 @@
+# Contributor: Dmitry Romanenko <dmitry@romanenko.in>
+# Maintainer: Dmitry Romanenko <dmitry@romanenko.in>
+pkgname=pytest-xdist
+_pkgname=pytest-xdist
+pkgver=1.26.0
+pkgrel=0
+pkgdesc="pytest xdist plugin for distributed testing and loop-on-failing modes"
+url="https://github.com/pytest-dev/pytest-xdist"
+arch="noarch"
+license="MIT"
+#options="!check" # Tests are outdated
+depends="py-execnet pytest pytest-forked py-six"
+checkdepends="py-py pytest py-tox"
+makedepends="python2-dev python3-dev py-setuptools py-setuptools_scm"
+subpackages="py3-$_pkgname:_py3 py2-$_pkgname:_py2"
+source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
+builddir="$srcdir"/$_pkgname-$pkgver
+
+build() {
+	cd "$builddir"
+	python2 setup.py build
+	python3 setup.py build
+}
+
+check() {
+	cd "$builddir"
+	tox -e py27,py36
```

Comment:
Probably tox could be replaced with pytest

Reply:
Tried that, seems it works using "python2 -m pytest" but couple of tests are looking for filelock python dependency that is not part of alpine packages yet (huh).

Reply:
Got tests running, but also added checkdependency on #6142